receiver/scraper/helper/internal/metadata/generated_telemetry.go

```diff
@@ -6,6 +6,7 @@ import (
 	"errors"
 
 	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/noop"
```

Comment:
I'm confused why the newly generated code has the line.

Reply:
> I'm confused why the newly generated code has the line.

:thinking: I am not sure myself, it's probably not related to the changes on this PR, so for now I will take a look but for now you can keep on working on the stuff I mentioned below